### PR TITLE
IDEX-3853 Git Outputs to be displayed in Consoles panel

### PR DESCRIPTION
**Pull request by RomanNikitenko**
Git Outputs to be displayed in Consoles panel

### CODE REVIEW

File: plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java [hunk 1]
```diff
@@ -325,6 +327,22 @@ public void addCommand(@NotNull String machineId, @NotNull final CommandConfigur
         }
     }
 
+    public void addGitCommand(@NotNull String machineId, @NotNull String commandName, @NotNull OutputConsole outputConsole) {
```
Comment (azatsarynnyy) [inline plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java]: javadoc

File: plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java [hunk 2]
```diff
@@ -299,8 +302,7 @@ public void apply(PromiseError arg) throws OperationException {
      *         the console for command output
      */
```
Comment (azatsarynnyy) [inline plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java]: please, remove this extra line between javadoc and addCommand method

File: plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java (hunk 1 above)
Comment (RomanNikitenko) [inline plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java]: Done

File: plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java (hunk 2 above)
Comment (RomanNikitenko) [inline plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java]: done

File: plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/branch/BranchPresenter.java [hunk 3]
```diff
@@ -271,12 +285,10 @@ private void printGitMessage(String messageText) {
             message = jsonObject.get("message").isString().stringValue();
         }
 
-        workspaceAgent.openPart(gitConsole, PartStackType.INFORMATION);
-
-        gitConsole.print("");
+        console.print("");
```
Comment (vparfonov) [inline plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/branch/BranchPresenter.java]: why we print empty string?

File: plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ProcessTreeRenderer.java [hunk 4]
```diff
@@ -70,6 +70,8 @@ public SpanElement renderNodeContents(ProcessTreeNode node) {
                 return createCommandElement(node);
             case TERMINAL_NODE:
                 return createTerminalElement(node);
+            case GIT_NODE:
```
Comment (vparfonov) [inline plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ProcessTreeRenderer.java]: not good hardcode git here

**Comment by vparfonov** [comment]
ok

**Pull request merged**